### PR TITLE
Entity Page - Cavatica Button (WIP until data issue fixed)

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

File: src/components/EntityPage/File/index.js [hunk 1]
```diff
@@ -339,19 +338,22 @@ const FileEntity = ({ api, fileId }) => {
...
-              <EntityActionBar>Share Button</EntityActionBar>
+              <EntityActionBar>
+                <CavaticaAnalyse fileId={fileId} />
+              </EntityActionBar>
               <EntityContent>
                 <EntityContentSection title="File Properties">
                   <Row style={{ width: '100%' }}>
                     <Column style={{ flex: 1, paddingRight: 15, border: 1 }}>
-                      <SummaryTable rows={table1} />
+                      {/* <SummaryTable rows={table1} /> */}
```
Comment (nyanofthemoon) [inline src/components/EntityPage/File/index.js]: Remove rather than comment-out?

File: src/components/EntityPage/File/index.js [hunk 2]
```diff
@@ -339,19 +338,22 @@ const FileEntity = ({ api, fileId }) => {
...
                     </Column>
                     <Column style={{ flex: 1, paddingLeft: 15, border: 1 }}>
-                      <SummaryTable rows={table2} />
+                      {/*<SummaryTable rows={table2} />*/}
```
Comment (nyanofthemoon) [inline src/components/EntityPage/File/index.js]: Remove rather than comment-out?

File: src/components/EntityPage/File/index.js [hunk 3]
```diff
@@ -339,19 +338,22 @@ const FileEntity = ({ api, fileId }) => {
...
                     </Column>
                   </Row>
                 </EntityContentSection>
                 <EntityContentDivider />
+                {/*
```
Comment (nyanofthemoon) [inline src/components/EntityPage/File/index.js]: Remove rather than comment-out?

File: src/components/EntityPage/File/index.js [hunk 4]
```diff
@@ -323,12 +323,11 @@ const FileEntity = ({ api, fileId }) => {
       {file => {
         if (file.isLoading) {
           return <div>Loading</div>;
-        } else if (!file.data) {
-          return <div>no data</div>;
```
Comment (nyanofthemoon) [inline src/components/EntityPage/File/index.js]: Do we want something else for the no data information?

**Comment by ciaranschutte** [comment]
^ @nyanofthemoon re all comments.
the base branch has the tables etc in it already, commented out here so it was working while developing, will be fixed in merge (i'll rebase before)

File: src/components/EntityPage/File/index.js (hunk 4 above)
Comment (ciaranschutte) [reply src/components/EntityPage/File/index.js]: Probably I'll check with Rosi

**Pull request merged**